src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.Android.cs

```diff
@@ -1,31 +1,88 @@
 ﻿#if __ANDROID__
+using System;
 using Android.App;
 using Android.Content;
+using Android.Content.Res;
 using Android.Hardware;
 using Android.Runtime;
+using Android.Views;
+using Uno.UI;
+using Android.Provider;
+using static Android.Provider.Settings;
+using Android.Database;
+using Android.OS;
 
 namespace Windows.Devices.Sensors
 {
 	public partial class SimpleOrientationSensor : Java.Lang.Object, ISensorEventListener
 	{
+		#region Static
+
+		private static Orientation _defaultDeviceOrientation;
+
+		private static Orientation DefaultDeviceOrientation
+		{
+			get
+			{
+				if (_defaultDeviceOrientation == Orientation.Undefined)
+				{
+					var context = ContextHelper.Current;
+					var windowManager = context.GetSystemService(Context.WindowService).JavaCast<IWindowManager>();
+					var config = context.Resources.Configuration;
+					var rotation = windowManager.DefaultDisplay.Rotation;
+
+					_defaultDeviceOrientation =
+						((rotation == SurfaceOrientation.Rotation0 || rotation == SurfaceOrientation.Rotation180) && config.Orientation == Orientation.Landscape) ||
+						((rotation == SurfaceOrientation.Rotation90 || rotation == SurfaceOrientation.Rotation270) && config.Orientation == Orientation.Portrait)
+							? Orientation.Landscape
+							: Orientation.Portrait;
+				}
+
+				return _defaultDeviceOrientation;
+			}
+		}
+
+		#endregion
+
+		private SimpleOrientationEventListener _orientationListener;
+		private SettingsContentObserver _contentObserver;
 		private SensorManager _sensorManager;
 		// Threshold, in meters per second squared, closely equivalent to an angle of 25 degrees which correspond to the value when Android detect new screen orientation 
 		private const double _threshold = 4.55;
-		private const Android.Hardware.SensorType _sensorType = Android.Hardware.SensorType.Gravity;
+		private const Android.Hardware.SensorType _gravitySensorType = Android.Hardware.SensorType.Gravity;
 
 		partial void Initialize()
 		{
 			_sensorManager = (SensorManager)Application.Context.GetSystemService(Context.SensorService);
-			_sensorManager.RegisterListener(this, _sensorManager.GetDefaultSensor(_sensorType), SensorDelay.Normal);
+			var gravitySensor = _sensorManager.GetDefaultSensor(Android.Hardware.SensorType.Gravity);
```

Comment:
Should use _gravitysensortype here?

Reply:
Oh yeah forgot to use it here, thanks will update ;) !